src/main/java/sirius/web/dispatch/HelpDispatcher.java

```diff
@@ -38,14 +35,33 @@ public boolean preDispatch(WebContext ctx) throws Exception {
     }
 
     @Override
+    public int getPriority() {
+        return PriorityCollector.DEFAULT_PRIORITY + 100;
+    }
+
+    @Override
     public boolean dispatch(WebContext ctx) throws Exception {
         if (!ctx.getRequest().uri().startsWith("/help") || HttpMethod.GET != ctx.getRequest().method()) {
             return false;
         }
         String uri = ctx.getRequestedURI();
+        String helpSystemlanguage = "";
         if ("/help".equals(uri) || "/help/".equals(uri)) {
             uri = "/help/" + indexTemplate;
         }
+        for (String language : getLanguages()) {
```

Comment:
Wouldn't a regex here be easier / nicer? Otherwise (or always) move language detection into its own method